panel_view.py

```diff
@@ -44,8 +44,9 @@ def plugin_unloaded():
 @events.on(events.LINT_RESULT)
 def on_lint_result(buffer_id, **kwargs):
     for window in sublime.windows():
-        if buffer_id in buffer_ids_per_window(window):
-            fill_panel(window, update=True)
+        if window.find_output_panel(PANEL_NAME):
+            if buffer_id in buffer_ids_per_window(window):
+                fill_panel(window, update=True)
```

Comment:
Instead of the nested if you could just AND both conditions.

Also maybe add a comment `don't fill the panel if it doesn't exist`.

Reply:
I think in one of my iterations it got a bit long so I split it. But that's better indeed.